README.md

```diff
@@ -20,14 +19,11 @@ jobs:
     permissions:
       checks: write # may not be necessary, see note below
     steps:
-      - uses: actions/checkout@v1
-      - name: Read ruby version
-        run: echo ::set-output name=RUBY_VERSION::$(cat .ruby-version | cut -f 1,2 -d .)
-        id: rv
-      - uses: actions/setup-ruby@v1
+      - uses: actions/checkout@v3
         with:
-          ruby-version: "${{ steps.rv.outputs.RUBY_VERSION }}"
-      - uses: planningcenter/balto-rubocop@v0.6
+          fetch-depth: 0
+      - uses: actions/setup-ruby@v1
```

Comment:
This is probably for another PR, but the way I read these repos, this is an outdated action and will only support up to Ruby 2.7.

[ruby/setup-ruby](https://github.com/ruby/setup-ruby) replaces [actions/setup-ruby](https://github.com/actions/setup-ruby) and includes newer Ruby versions.

Reply:
Good catch! This is exactly the PR to do it it, especially since the bit I removed I think can only be done with this newer version.  😬